zk/src/archive/web/js/zk/au.js

```diff
@@ -1505,6 +1506,7 @@ zAu.cmd0 = /*prototype*/ { //no uuid at all
 					message: msg
 				});
 			});
+			jq(w).on('keydown', zk.$void);
```

Comment:
Use two anchors around w, and send tab (or shift+tab) to the next focusable element